advocacy_docs/partner_docs/ImpervaDataSecurityFabric/04-ConfiguringImpervaDataSecurityFabric.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Configuring Imperva Data Security Fabric Agent'
```

Comment:
Can we leave Agent out of this?

Reply:
No because this is the actual agent portion that the configuration is for.  This is how they have it and I cleaned it up.